src/daemon/rpmostreed-daemon.c

```diff
@@ -180,11 +207,26 @@ on_active_txn_changed (GObject *object,
       g_variant_get (active_txn, "(&s&s&s)", &method, &sender, &path);
       if (*method)
         {
-          gboolean is_registered = rpmostreed_daemon_has_client (self, sender);
-          const char *caller_str = is_registered ? "client" : "caller";
-          sd_journal_send ("MESSAGE_ID=" SD_ID128_FORMAT_STR, SD_ID128_FORMAT_VAL(RPMOSTREE_MESSAGE_TRANSACTION_STARTED),
-                           "MESSAGE=Initiated txn %s for %s %s: %s", method, caller_str, sender, path,
-                           NULL);
+          g_autofree char *client_str = rpmostreed_daemon_client_get_string (self, sender);
+          struct RpmOstreeClient *clientdata = g_hash_table_lookup (self->bus_clients, sender);
+          if (!clientdata)
+            {
+              sd_journal_send ("MESSAGE_ID=" SD_ID128_FORMAT_STR, SD_ID128_FORMAT_VAL(RPMOSTREE_MESSAGE_TRANSACTION_STARTED),
+                               "MESSAGE=Initiated txn %s for %s: %s", method, client_str, path,
+                               "BUS_ADDRESS=%s", sender,
+                               NULL);
+            }
+          else
+            {
+              g_autofree char *client_data_msg = NULL;
+              if (clientdata->uid_valid)
+                client_data_msg = g_strdup_printf ("CLIENT_UID=%u", clientdata->uid);
+              sd_journal_send ("MESSAGE_ID=" SD_ID128_FORMAT_STR, SD_ID128_FORMAT_VAL(RPMOSTREE_MESSAGE_TRANSACTION_STARTED),
+                               "MESSAGE=Initiated txn %s for %s: %s", method, client_str, path,
+                               "BUS_ADDRESS=%s", sender,
+                               client_data_msg ?: NULL,
+                               NULL);
```

Comment:
I started that originally but I changed it...I forget exactly why.  I think it had something to do with that approach not generalizing to *multiple* optional keys (since the va_arg processing would stop at the first `NULL`).  Fixing that would require some sort of "builder api" style logging.

In this case though we don't have multiple keys, so I changed it back.